sdk/maps/Azure.ResourceManager.Maps/tests/ScenarioTests/MapsAccountTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Maps.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Maps.Tests
+{
+    public class MapsAccountTests : MapsManagementTestBase
+    {
+        public MapsAccountTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private ResourceGroupResource ResourceGroup { get; set; }
+
+        private MapsAccountCollection MapCollection { get; set; }
+
+        private async Task SetCollection()
+        {
+            ResourceGroup = await CreateResourceGroupAsync();
+            MapCollection = ResourceGroup.GetMapsAccounts();
+        }
+
+        [Test]
+        public async Task MapsAccountCreateTest()
+        {
+            await SetCollection();
```

Comment:
actually we could simply do this here
```suggestion
            var ResourceGroup = await CreateResourceGroupAsync();
            var MapCollection = ResourceGroup.GetMapsAccounts();
```
therefore we no longer need these variables as properties

Reply:
Fixed